vendor/engines/bulk_email/db/migrate/20161024213800_create_bulk_email_jobs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBulkEmailJobs < ActiveRecord::Migration
+
+  def change
+    create_table :bulk_email_jobs do |t|
+      t.references :facility, null: false, foreign_key: true
```

Comment:
Let's make sure to double check that this actually adds the regular index in Oracle. It's weird about creating a foreign key does not implicitly create an index like it does in MySQL.